rocAL/rocAL/source/pipeline/ring_buffer.cpp

```diff
@@ -339,6 +352,10 @@ void RingBuffer::release_gpu_res()
                         //printf("Error Freeing device buffer <%d, %d, %p>\n", buffIdx, sub_buf_idx, _dev_sub_buffer[buffIdx][sub_buf_idx]);
                         ERR("Could not release hip memory in the ring buffer")
                     }
+                if (_dev_roi_buffers[buffIdx][sub_buf_idx])
+                    if ( hipHostFree((void *)_dev_roi_buffers[buffIdx][sub_buf_idx]) != hipSuccess ) {
+                        ERR("Could not release hip memory for ROI in the ring buffer")
+                    }
```

Comment:
Add {} after if condition since theres a check of more than 1 line after the if condition

Reply:
done